src/main/java/com/mattworzala/canary/server/command/test/BuilderCommand.java

```diff
@@ -32,27 +35,80 @@
 import java.util.concurrent.CountDownLatch;
 
 import static com.mattworzala.canary.server.command.TestCommand.version;
+import static net.minestom.server.command.builder.arguments.ArgumentType.Literal;
 
 public class BuilderCommand extends Command {
     private static final String NAME = "builder";
     private static final String VERSION = "0.0.1";
 
     private ItemStack testBuilderItem;
+    private TestBuilderController testBuilderController;
+    private boolean isPlayerInTestBuilder = false;
```

Comment:
A solution like this excludes multiple players from being in a test builder (which seems like a potentially reasonable thing to do in the future?